gen/libclang_api.jl

```diff
@@ -954,7 +954,7 @@ function clang_getIBOutletCollectionType(arg1)
 end
 
 function clang_visitChildren(parent, visitor, client_data)
-    ccall((:clang_visitChildren, libclang), UInt32, (CXCursor, CXCursorVisitor, CXClientData), parent, visitor, client_data)
```

Comment:
This changes the API, which is why the version I wrote inlined the `ccall`. This is certainly fine if this is an internal API and you've updated all the callers.

You can also check if other ones has similar pattern.

FWIW, this seems like a very thin wrapper and putting the ccall inline doesn't seem too bad to me...... just personal taste biased on the readability for ppl that didn't write the codde...